packages/map3d/src/interfaces/props.interface.ts

```diff
@@ -0,0 +1,9 @@
+import {PlanetInterface} from './planet.interface';
+
+export interface Map3dPropsInterface {
+  currentUserId: string | undefined;
```

Comment:
Are you sure not: 
```suggestion
  currentUserId?: string;
```
?